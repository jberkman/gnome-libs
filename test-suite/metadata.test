the_test=metadata
. ../../../test-suite/runtest.sh
