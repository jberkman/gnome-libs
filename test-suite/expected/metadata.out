vec has 2 elements
alpha.txt has widdershins
foo.txt has yadda
foo.joe has yadda
zoe.joe has xenophobe
now vec has 1 elements
bar has yadda
now foo.txt has widdershins
